package-lock.json

```diff
@@ -3361,4 +3361,4 @@
       "dev": true
     }
   }
-}
+}
```

Comment:
Please add a newline at the end of the file. Don't know why this happened but that would be great.